config/install/core.entity_view_display.user.user.full.yml

```diff
@@ -0,0 +1,75 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.user.full
+    - field.field.user.user.field_admin_details
+    - field.field.user.user.field_last_password_reset
+    - field.field.user.user.field_location_details
+    - field.field.user.user.field_password_expiration
+    - field.field.user.user.field_pending_expire_sent
+    - field.field.user.user.field_phone_number
+    - field.field.user.user.field_school_details
+    - views.view.school_basic_information
+    - views.view.user_basic_information
+  module:
+    - layout_builder
+    - layout_discovery
+    - user
+    - views
+third_party_settings:
+  layout_builder:
+    enabled: true
+    allow_custom: false
+    sections:
+      -
+        layout_id: layout_onecol
+        layout_settings:
+          label: ''
+          context_mapping: {  }
+        components:
+          a6e915f0-ecf4-4f27-a9ac-509772a9875b:
+            uuid: a6e915f0-ecf4-4f27-a9ac-509772a9875b
```

Comment:
Not sure above this uuid, if this is required

Reply:
Yes, required.